core/arch/arm/kernel/user_ta.c

```diff
@@ -313,6 +313,14 @@ static TEE_Result ta_load(const TEE_UUID *uuid,
 		goto error_return;
 	}
 
+	if (ta_head->flags & TA_FLAG_MULTI_SESSION &&
```

Comment:
@jforissier  I have test the case TA_FLAG_INSTANCE_KEEP_ALIVE is set and TA_FLAG_SINGLE_INSTANCE not set,  when running the same TA, it will soon get to error message "Failed to allocate ASID", i think some resources are leaked .